CharacterCreation/03_Feats.txt

```diff
@@ -13,76 +13,76 @@ Passive Feats
 ==============================

```

Comment:
Can we add a title? Just calling it "Feats" would be helpful.

Reply:
Can we also have a sub-heading the same type as "Active Feats" and "Stat Feats" that says "Passive Feats"? That will make the table of contents make much more sense.

Reply:
I'm thoroughly confused:

![image](https://user-images.githubusercontent.com/2192156/33745266-a392eb9a-db6a-11e7-86ba-abc08d9159f9.png)

It sounds like you're asking me to put in "Passive Feats" but that already exists; what do you mean? There currently isn't a "Stat Feats" do you want me to add that in somewhere (I don't know it would make sense)?

Reply:
nope, my bad. Git online hides things after 5 unchanged lines in either direction and I just missed it. No change requested.